worlds/zork_grand_inquisitor/options.py

```diff
@@ -0,0 +1,61 @@
+from dataclasses import dataclass
+
+from Options import Choice, DefaultOnToggle, PerGameCommonOptions, Toggle
+
+
+class Goal(Choice):
+    """
+    Determines the victory condition
+
+    Three Artifacts: Retrieve the three artifacts of magic and place them in the walking castle
+    """
+    display_name: str = "Goal"
+
+    default: int = 0
+    option_three_artifacts: int = 0
```

Comment:
I have different goals planned for the future, so my code is already prepared for this eventuality by always treating the goal as a variable, even if there is only a single one for now. 

I agree that the user-facing consequence of doing this will seem odd. Let me think about it

Reply:
Thought about it. I hope we are ready to accept it as a quirk (I'm fairly certain I've seen other worlds have single choice options like that to be expansion friendly). I considered removing the option and passing a hardcoded value to `GameController` instead until there is at least a second goal, but unfortunately, it's not the only usage. The PopTracker pack also makes use of it, and honestly, I would rather not revisit that if possible. There's already more than 30 hours of (unenjoyable) work that went in that pack.

I could also add a note in the option docstring, but that may be awkward too. I don't get the feeling many users will care, and I'm not a fan of over-explaining / justifying things when it's not strictly necessary (i.e. here's why this option only has 1 value for now)